UnityProject/Assets/Scripts/MapSaver/MapSaver.cs

```diff
@@ -380,15 +380,15 @@ public int Compare(string x, string y)
 						{
 							return y1.CompareTo(y2);
 						}
-						else if (z1 != z2)
+						else if (Mathf.Approximately(z1.Value, z2.Value) == false)
 						{
 							return z1.Value.CompareTo(z2.Value);
 						}
 						else
 						{
 							var guidx = x.Split('@')[1];
 							var guidy = y.Split('@')[1];
-							return guidx.CompareTo(guidy);
+							return String.Compare(guidx, guidy, StringComparison.Ordinal);
```

Comment:
there's a setting globally that does this as well

Reply:
where?

Reply:
sorry it's localisation stuff was confused

		// Set the global culture to InvariantCulture
			CultureInfo.DefaultThreadCurrentCulture = CultureInfo.InvariantCulture;
			CultureInfo.DefaultThreadCurrentUICulture = CultureInfo.InvariantCulture;